pallets/xor-fee/src/lib.rs

```diff
@@ -623,7 +623,7 @@ pub mod pallet {
         type ReferrerWeight: Get<u32>;
         type XorBurnedWeight: Get<u32>;
         type XorIntoValBurnedWeight: Get<u32>;
-        type BuyBackXSTPercent: Get<Percent>;
+        type GetBuyBackXSTPercent: Get<Percent>;
```

Comment:
Maybe also rename other `Get<_>` associated types? The naming seems to become inconsistent

Reply:
or not rename it at all